tweepy/streaming.py

```diff
@@ -168,7 +170,7 @@ def read_line(self, sep='\n'):
                 return self._pop(loc + len(sep))
             else:
                 start = len(self._buffer)
-            self._buffer += self._stream.read(self._chunk_size)
+            self._buffer += self._stream.read(self._chunk_size).decode(self._encoding)
```

Comment:
I'm not sure it is safe to perform decoding on arbitrary chunks of data as they come in as you could end up in a situation where you are using a valid decode on incomplete data.  Take for instance the following (python3.4):

```
>>> skull = "☠"
>>> skull_encoded = skull.encode('utf-8')
>>> skull_encoded
b'\xe2\x98\xa0'
>>> skull_encoded[:-1].decode('utf-8')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'utf-8' codec can't decode bytes in position 0-1: unexpected end of data
```